.github/workflows/auto-merge.yml

```diff
@@ -23,8 +23,9 @@ jobs:
           app-id: ${{ secrets.MERGE_APP_ID }}
           private-key: ${{ secrets.MERGE_APP_KEY }}
       - name: Set auto merge
-        uses: paritytech/auto-merge-bot@v1.0.1
+        uses: paritytech/auto-merge-bot@v1.1.0
         with:
           GITHUB_TOKEN: ${{ steps.merge_token.outputs.token }}
           MERGE_METHOD: "SQUASH"
           ALLOWLIST: ${{ steps.fellows.outputs.github-handles }}
+          UPDATE_BEFORE_MERGE: true
```

Comment:
Otherwise the "automatic merge" it not triggered any way.

Reply:
I mean, it was doing this before already or not?

Reply:
It wasn't doing this before.

The new version runs the [`updatePullRequestBranch`](https://docs.github.com/en/graphql/reference/mutations#updatepullrequestbranch) mutation before enabling auto-merge if this field is enabled.

Apologizes for the delay of the feature, but now it should work as intended and always update the PR before enabling auto-merge.

Reply:
Nice!